tasks/install.yml

```diff
@@ -0,0 +1,68 @@
+---
+- import_tasks: apt_install.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add gitlab user
+  user:
+      name: git
+      shell: /bin/nologin
+      comment: 'GitLab'
+      state: present
+      groups: redis
```

Comment:
What about the user being a system user?

Reply:
fixed